assets/js/sticky-menu.js

```diff
@@ -136,16 +137,23 @@ define( ['jquery', 'underscore'], function ( $, _ ) {
 		 * Get the direction of scroll (negative value = up, positive value = down).
 		 */
 		getScrollDirection: function () {
-			var currentWindowTop = $( window ).scrollTop(),
-					value            = currentWindowTop - this.windowTop;
+			var currentWindowTop = $(window).scrollTop(),
+				value = currentWindowTop - this.windowTop;
 
 			this.windowTop = currentWindowTop;
 
 			return value;
 		},
 
 		/**
-		 * Is the direction of the scroll = up?
+		 * Is scroll all directions enabled?
+		 */
+		isAllDirectionEnabled: function () {
+			return $( 'body' ).hasClass( config.bodyStickyVisibilityClass );
+		},
+
+		/**
+		 * Is the direction of the scroll = up or down?
```

Comment:
@krumpak this comment is not OK. The words "or down" does not apply here, because the method return true only, if the scroll is "up" and not if it's "down". So the "or down" should be removed.